src/libstore/daemon.cc

```diff
@@ -222,7 +222,8 @@ struct ClientSettings
                     else if (!hasSuffix(s, "/") && trusted.count(s + "/"))
                         subs.push_back(s + "/");
                     else
-                        warn("ignoring untrusted substituter '%s'", s);
+                        warn("ignoring untrusted substituter '%s', you are not a trusted user.\n"
+                             "More information about 'trusted-substituters' option in nix.conf man page", s);
```

Comment:
```suggestion
                             "More information about 'trusted-substituters' option can be found in the nix.conf man page or at \nhttps://nixos.org/manual/nix/stable/command-ref/conf-file.html#conf-trusted-substituters", s);
```

Reply:
@eordano we [discussed this recently](https://github.com/NixOS/nix/issues/6672#issuecomment-1357976553) and decided not to add a URL, because that would be in there forever, while quite literally everything about that URL may change in the future. Only the man pages are guaranteed to be in sync with the code.

Reply:
I've suggested a new change then, yes, I also felt strange about `stable` -- having to detect what channel the user is using could be complex.